includes/wikia/parser/templatetypes/tests/NavigationTemplateTest.php

```diff
@@ -35,7 +36,7 @@ public function getNavigationTemplates() {
 		return [
 			[
 				'<a>This is a <strong>template</strong> <b>without</b> a <span>block</span> element</a>.',
-				'<a>This is a <strong>template</strong> <b>without</b> a <span>block</span> element</a>.',
+				"\n<a>This is a <strong>template</strong> <b>without</b> a <span>block</span> element</a>.\n",
```

Comment:
this is side effect of the fix, we should decide if it is ok or we should fix it somehow.